### PR TITLE
[Claim] Profile: vCOW balance

### LINKED ISSUE #2007: Add vCOW balance to Profile
**Description**

Add vCOW balance to Profile

Crude wireframe:
Screenshot from 2021-12-17 10-41-34

**Proposal**

Split page into 2 sections:
1. Profile: containing only `vCOW` balance for the connected network
2. Affiliate Program: what we already have

When no `vCOW` balance, don't show anything on the `Profile` section

**Good to have**
When there's a vesting, show a dropdown with additional details
Screenshot from 2021-12-17 10-46-15

**Additional context**
- Data will be mocked for now, later they will read from a hook we provide
- While only the mocked data is available, show it only on dev, prs and localhost. Do not show on staging, barn nor prod. (There are flags in the code to identify the current environment)
- Wireframes: https://www.figma.com/file/DPp3awAUcqpwpV1dD4wFB9/CoW-claiming?node-id=33%3A4161

### PR DESCRIPTION
**Pull request by alongoni**
# Summary
Fixes #2007 
Dropdown will be implemented in an other PR when we have all the data ready.
image
image

Before (dark backround on both themes):
image

  # To Test

1. Open the page [`Profile`](https://pr2051--gpswapui.review.gnosisdev.com/#/profile)

### CODE REVIEW

File: src/custom/pages/Profile/index.tsx [hunk 1]
```diff
@@ -28,6 +32,8 @@ import NotificationBanner from 'components/NotificationBanner'
 import { SupportedChainId as ChainId } from 'constants/chains'
 import AffiliateStatusCheck from 'components/AffiliateStatusCheck'
 import { useHasOrders } from 'api/gnosisProtocol/hooks'
+import CowProtocolLogo from '@src/custom/components/CowProtocolLogo'
+import { Title } from '@src/custom/components/Page'
```
Comment (matextrem) [inline src/custom/pages/Profile/index.tsx:31]: Path should start from `components/`

**Review by matextrem — CHANGES_REQUESTED** [review]

**Comment by alfetopito** [comment]
Tiny question: why the different font size on both sections?
Screen Shot 2022-01-06 at 09 15 31

Although Profile is the page name, the two sections are equivalent IMO and should have the same font size on the headers

**Review by alfetopito — APPROVED** [review]
Other than the font size comment, looks good to me

**Review by anxolin — APPROVED** [review]
Looks ok to me, happy to merge this (or continue with the task of getting the real value and formatting the amount)

One minor concern to discuss with @michelbio is regarding the colors. I'm not sure the black background

One small thing we can do to make it look better imo is just use sth similar to what we used in the profile:
image

**Comment by alongoni** [comment]
> Tiny question: why the different font size on both sections? Screen Shot 2022-01-06 at 09 15 31
> 
> Although Profile is the page name, the two sections are equivalent IMO and should have the same font size on the headers

Good point. I was thinking we should have a main title as we have in other sections (e.g about, faqs). But anyway we can use the same font-size in both cases.

**Comment by alfetopito** [comment]
> Looks ok to me, happy to merge this (or continue with the task of getting the real value and formatting the amount)
> 
> One minor concern to discuss with @michelbio is regarding the colors. I'm not sure the black background
> 
> One small thing we can do to make it look better imo is just use sth similar to what we used in the affiliate part: image

Indeed, we talked about this during the Protofire sync.
What we agreed on was to use the light/dark colors as a start, and iterate with Michel's input when he's back

But of course, open for feedback already

**Review by matextrem — APPROVED** [review]

**Comment by alongoni** [comment]
> Looks ok to me, happy to merge this (or continue with the task of getting the real value and formatting the amount)
> 
> One minor concern to discuss with @michelbio is regarding the colors. I'm not sure the black background
> 
> One small thing we can do to make it look better imo is just use sth similar to what we used in the affiliate part: image

Hey @anxolin, thanks! Makes sense! Now we are using the same color than the affiliate section. It'll be changed when we have the dropdown working. 
image

**Pull request merged**